blog/what-is-source-code-verification/index.mdx

```diff
@@ -0,0 +1,733 @@
+```
+
+Onchain runtime bytecode:
+```
+0x7399f7c0086ab897c8fe65ebff268c732a7e15b25f3014608060405260043610610034575f3560e01c8063cad0899b14610038575b5f80fd5b610052600480360381019061004d91906100b4565b610068565b60405161005f9190610101565b60405180910390f35b5f81836100759190610147565b905092915050565b5f80fd5b5f819050919050565b61009381610081565b811461009d575f80fd5b50565b5f813590506100ae8161008a565b92915050565b5f80604083850312156100ca576100c961007d565b5b5f6100d7858286016100a0565b92505060206100e8858286016100a0565b9150509250929050565b6100fb81610081565b82525050565b5f6020820190506101145f8301846100f2565b92915050565b7f4e487b71000000000000000000000000000000000000000000000000000000005f52601160045260245ffd5b5f61015182610081565b915061015c83610081565b92508282019050808211156101745761017361011a565b5b9291505056fea26469706673582212209d56669aec61ac1867a59a6d0472bd74975630174bc6e74d7e6881bba174c30564736f6c634300081a0033
+```
+
+</details>
+
+## After matching
+
+If we got a match, it means we can start from recompiled bytecodes, apply each runtime or creation transformations on top of the respective recompiled bytecodes, and obtain the onchain bytecodes. It also means that the provided source code and metadata can be associated with the contract at this chainId and address and we mark this contract as verified.
+
+Finally, we store our contract in our storage backend of choice. Currently this can be a file system or a SQL database in Sourcify's case. By default we do both. See the [Contract Repository docs](https://docs.sourcify.dev/docs/repository/) for more information.
+
+## EOF and the road ahead
+
+Big changes to how the EVM bytecode is structured is coming up with the [EVM Object Format](https://evmobjectformat.org/) proposal. According to how the compilers implement these changes, the verification process face changes and verifying pre-EOF contracts will be different than the EOF contracts. 
+
+One nice feature of the EOF is it separates code from data and gives structure to code. This makes us verifiers' lives much easier compared to the completely unstructured legacy EVM code. A current example can be seen in our post [Finding Auxdatas in the Bytecode](/blog/finding-auxdatas-in-bytecode/). However as of now, the metadata is still not fully isolated from the contract code as there is no separate section for that information. Current compiler implementations put the metadata in the `data_section` of EOF and this unfortunately does affect the contract's code. That's why we are proposing a separate `metadata_section` in the EVM Object Format that cannot be reached by the EVM and any changes to that will not affect the contract's code. If this is implemented, verifiers can completely ignore the metadata parts of a container, and won't have to look for workarounds.
```

Comment:
> However as of now,

Here I would specify that "now" it means the current EOF specification

Reply:
> verifiers can completely ignore the metadata parts of a container

Also "data section", right?

Reply:
I think so too that we can ignore `data_section` but I'm not 100% sure so I'd refrain from saying that